api/prisma/schema.prisma

```diff
@@ -9,3 +9,21 @@ datasource db {
   provider = "mysql"
   url      = env("DATABASE_URL")
 }
+
+model Deal {
+  dealId Int @id @default(autoincrement())
+  business @relation(fields: [dealId], references: [businessId])
+  service @relation(fields: [dealId], references: [serviceId])
```

Comment:
service should be:
`service @relation(fields: [serviceID], references: [serviceID])`